server/pbenchinacan/etc/pbench-server/pbench-server.cfg

```diff
@@ -27,9 +27,11 @@ port = 9200
 [database]
 uri = postgresql://pbenchcontainer:pbench@pbenchinacan:5432/pbenchcontainer
 
+[authentication]
+secret-key = "pbench-in-a-can secret shhh"
+
 # User authentication section to use when authorizing the user with an OIDC
 # identity provider.
-[authentication]
 
 # URL of the OIDC auth server
 server_url = http://pbenchinacan:8090
```

Comment:
Line 31 should have been inserted after line 35.  (Or, lines 33-34 should be moved before line 30....)

Reply:
Fixed.